unittests/integration/builder_tester.cpp

```diff
@@ -63,14 +63,14 @@ CUDAQ_TEST(BuilderTester, checkSimple) {
     };
     cudaq::gradients::central_difference gradient(ansatz, argMapper);
     cudaq::optimizers::lbfgs optimizer;
-    optimizer.initial_parameters = {0.35, 0.25};
-    optimizer.max_eval = 10;
-    optimizer.max_line_search_trials = 10;
+    optimizer.initial_parameters = {0.359, 0.257};
+    optimizer.max_eval = 4;
+    optimizer.max_line_search_trials = 8;
     auto [opt_val_0, optpp] =
         cudaq::vqe(ansatz, gradient, h3, optimizer, 2, argMapper);
     printf("Opt-params: %lf %lf \n", optpp[0], optpp[1]);
     printf("<H3> = %lf\n", opt_val_0);
-    EXPECT_NEAR(opt_val_0, -2.045375, 1e-3);
+    EXPECT_NEAR(opt_val_0, -2.045375, 1e-2);
```

Comment:
If we made the initial parameters more accurate, why did we have to make the tolerance larger?

Reply:
We need these 3 things altogether: reduce the number of optimization runs (line search and iterations), better initial values, and relax the test conditions. If preferred, I'll restore the tolerance and increase the optimizer parameters a bit.

Reply:
Was this particular change to reduce the test runtime?

What would you think about updating  the `-2.045375` to a new value and leaving `1e-3`?

Reply:
If this is really just to reduce test runtime, and if -2.045375 is truly the "correct" answer, then I suppose I'm fine leaving it the way it is.

Reply:
Yes, it's really just to reduce the test runtime. I think I'd go with keeping the true value here for future references.